README.md

```diff
@@ -2642,6 +2643,58 @@ Produces a random number between the inclusive lower and upper bounds. If only o
 
 **[⬆ back to top](#quick-links)**
 
+### _.clamp
+
+Clamps number within the inclusive lower and upper bounds.
+
+```js
+// Lodash
+_.clamp(-10, -5, 5);
+// => -5
+ 
+_.clamp(10, -5, 5);
+// => 5
+
+_.clamp(10, -5);
+// => -5
+
+_.clamp(10, 99);
+// => 10
+
+// Native
+const clamp = (number, boundOne, boundTwo) => {
+  if (!boundTwo) {
+    return Math.max(number, boundOne) === boundOne ? number : boundOne; 
+  } else if (Math.min(number, boundOne) === number) {
```

Comment:
- If the number is greater than the upper bound, the upper bound is return; likewise if the number is less than the lower bound, the lower bound is returned
- If nothing has been returned from those checks then that means the number does not need to be clamped as it falls within the bounds and the number itself is returned